pages/index.tsx

```diff
@@ -1,14 +1,21 @@
+import { useUser } from '@auth0/nextjs-auth0'
 import React from 'react'
 
 import BaseLayout from '../components/common/BaseLayout'
+import Loader from '../components/common/Loader'
 import MainContent from '../components/home/MainContent'
 
 export default function Home() {
-  return (
-    <div>
-      <MainContent />
-    </div>
-  )
+  const { user, error, isLoading } = useUser()
+
+  if (isLoading || error)
+    return (
+      <div>
+        <Loader open={isLoading} error={error} />
+      </div>
+    )
+
+  return <div>{user ? null : <MainContent />}</div>
```

Comment:
if the user is logged in then take to the dashboard page instead of returning null

Reply:
fixed